src/app/shared/components/template/components/button/button.component.html

```diff
@@ -25,16 +25,18 @@
         [attr.data-rowname]="_row.name"
       ></plh-template-component>
     </span>
+    <ion-ripple-effect></ion-ripple-effect>
   </ion-button>
 
   <!-- "card-portrait" variant. This variant is not achievable using an ion-button -->
   <div
     *ngSwitchCase="true"
-    class="button-container"
+    class="button-container ion-activatable"
     (click)="triggerActions('click')"
     [attr.data-variant]="params.variant"
     [attr.data-has-children]="_row.rows ? true : false"
   >
+    <ion-ripple-effect type="unbound"></ion-ripple-effect>
```

Comment:
From the [docs](https://ionicframework.com/docs/api/ripple-effect#type), it looks like the two types are  `"bounded"` and `"unbounded"`, so `"unbound"` will have no effect. I don't think we want the effect to be unbounded, so you can remove this `type` attribute.

Reply:
Will do!